Doc/source/designspaceLib/index.rst

```diff
@@ -1,16 +1,16 @@
-##############
-designspaceLib
-##############
+########################################################
+designspaceLib: Read, write, and edit .designspace files
```

Comment:
Elsewhere in the documentation they are called “designspace files”, without the dot.